styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/MeteredProxy.java

```diff
@@ -23,13 +23,16 @@
 import com.spotify.styx.docker.DockerRunner;
 import com.spotify.styx.storage.Storage;
 import com.spotify.styx.util.Time;
+import io.fabric8.kubernetes.client.KubernetesClientException;
```

Comment:
This kind of dependencies feels weird to be in the proxy. But don't have a better solution. Maybe we can have a separated class handling reporting of failed operations (not limited to k8s in the future).

Reply:
Yeah, I would rather wrap the k8s client directly, e.g. in the `KubernetesDockerRunner` but I wanted to get something in place quickly and the `MeteredProxy` seemed like a convenient place to quickly get some metrics into place with a minimal amount of code.

Reply:
If the TODO will be addressed later (not staying forever:), ok for me.